pkg/console/operator/operator.go

```diff
@@ -140,15 +140,7 @@ func NewConsoleOperator(
 
 // key is actually the pivot point for the operator, which is our Console custom resource
 func (c *consoleOperator) Key() (metav1.Object, error) {
-	operatorConfig, err := c.operatorConfigClient.Get(api.ConfigResourceName, metav1.GetOptions{})
-	if errors.IsNotFound(err) && CreateDefaultConsoleFlag {
```

Comment:
`CreateDefaultConsoleFlag` is still referenced in a few other places

Reply:
Yup. Should be eliminated now.

Reply:
no longer in the help output:

```
console operator -h
An Operator for a web console for OpenShift.

Usage:
  console operator [flags]

Flags:
      --config string       Location of the master configuration file to run from.
  -h, --help                help for operator
      --kubeconfig string   Location of the master configuration file to run from.
```